paddle/phi/core/dense_tensor.inl

```diff
@@ -144,6 +152,7 @@ protected:
  */
 
 dnnl::memory::format_tag format_ = dnnl::memory::format_tag::undef;
+dnnl::memory::desc mem_desc_;
```

Comment:
Related discussion:
@chenwhql @Shixiaowei02 Do you think it would be better to move these two variables(format_ and mem_desc_) and all other member variables added in the future to dense_tensor.h? Since outer users can only use dense_tensor.h but not dense_tensor.inl and WITH_MKL will be always set ON when AVX is found when Paddle compilation, this will lead to inconsistent DenseTensor class between Paddle and outer users' plugins.